### DISCUSSION TITLE
How to reorder all columns alphabetically?

### DISCUSSION THREAD

**Discussion by ChillarAnand** [johnkerl/miller#1283] [Q&A] [answered]
There are few softwares which produce csv files and they change the order of the columns everytime. 

I am trying to reorder all the columns alphabetically so that it would be easier to diff when using other tools. 

The current reorder option is asking for a list of fields. Instead is there an option to sort all of them automatically?

**Comment by aborruso** [comment, marked as answer]
You can use [sort-within-records](https://miller.readthedocs.io/en/6.7.0/reference-verbs/index.html#sort-within-records)

```
mlr --csv sort-within-records input.csv
```
